BUILDING_ON_WINDOWS.md

```diff
@@ -84,7 +84,7 @@ Compiling with Breakpad support enables crash reports that can be of use for dev
 
 1. Open the `chatterino.pro` file by double-clicking it, or by opening it via Qt Creator.
 2. You will be presented with a screen that is titled "Configure Project". In this screen, you should have at least one option present ready to be configured, like this:
-   ![Qt Create Configure Project screenshot](https://user-images.githubusercontent.com/41973452/159462759-470e5371-671e-478e-85ca-33452ca9bea3.png)
+   ![Qt Create Configure Project screenshot](https://i.imgur.com/dbz45mB.png)
```

Comment:
1. The image should use githubusercontent (just paste it into a github comment and you can get a link
2. The image downgrades to VS 2017 and Qt 5.11, it should use one of our supported versions of Qt5 and VS 2022

Reply:
i can do it when i am back at home yes